packages/react/src/AnchorNav/AnchorNav.module.css

```diff
@@ -356,25 +361,40 @@
   align-self: flex-start;
 }
 
-@media screen and (max-width: 34rem) {
+@media screen and (max-width: 40rem) {
   .AnchorNav-action {
     display: none;
   }
+
+  .AnchorNav__actionsContainer {
+    flex-direction: column;
+  }
+
   .AnchorNav--expanded .AnchorNav-action {
     display: flex;
+    width: 100%;
     margin-top: var(--base-size-8);
   }
 }
 
-@media screen and (min-width: 34rem) and (max-width: 63.25rem) {
-  .AnchorNav-action {
-    display: flex;
+@media screen and (min-width: 40rem) and (max-width: 63.25rem) {
+  .AnchorNav__actionsContainer {
     position: absolute;
     top: 0;
     right: 0;
   }
 }
 
+@media screen and (min-width: 40rem) {
+  .AnchorNav__actionsContainer {
+    gap: var(--base-size-16);
+  }
+
+  .AnchorNav__actionsContainer--multiple {
+    margin-inline-start: var(--base-size-16);
```

Comment:
@natalie-iv  @auareyou FYI - Added a 16px gap when two actions are used, as the lack of available space otherwise pushes the links too close to the buttons.